source/features/useful-not-found-page.js

```diff
@@ -70,10 +71,13 @@ async function addDefaultBranchLink(bar) {
 	);
 }
 
-export default function () {
+features.add('useful-not-found-page', [
```

Comment:
I would make it accept only one argument object with various properties. Then it would be easy to add other things in the future, like a `description` field for showing the features in the settings UI, for example. I would also move the main function body to a top-level variable, which would look better.

Something like this:

```js
const feature = () => {
	…
};

features.add({
	id: 'useful-not-found-page',
	dependencies: [
		features.is404,
		features.domLoaded
	],
	feature
});
```

Not sure about the name `feature` for the feature function though.

Reply:
A bit verbose but I guess it won’t matter much with a little copy/paste